news_items/029_non_submittable/index.html

```diff
@@ -0,0 +1,79 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     width: 60%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+ .code-block div {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: -1em;
+     line-height: 125%;
+ }
+</style>
+
+<p>
+  Thanks to great work by 
+  <a href="https://github.com/echau01">Eugene Chau</a>, a student in
+  the University of British Columbia, it is now possible to allow
+  students to see assessments that are not available for submission,
```

Comment:
I feel like it's not really clear here that "not available for submission" also means "can't be started / can't be viewed". I wonder whether it's worth separating out the "before release data" and "after deadline" cases more, and explaining them each in more detail. For the "before release" case I also find that phrases like "allow students ... to see it" (from a later paragraph) is confusing. Could we be clearer about the two meanings of "see it": (1) the assessment shows up on the list of assessments, with future dates; and (2) being able to view the assessment, including all the questions. I'm thinking about something like:

> 1. Before an assessment has been released, using `"submittable": false` means the students will see the assessment on the course homepage, including the release date, but they won't be able to start the assessment or see any of the questions.
> 2. After students have completed an assessment, using `"submittable": false` means the students will be able to view their work, including seeing all questions, but they won't be able to submit any further answers. If they had not yet started the assessment by this time, then they won't be able to view it at all.

Trying to explain this makes me wonder whether there is a better word than `submittable`. Maybe `active`? Another possibility would be to separate out the `startAsessment` and `submitAnswers` restrictions. But I think that it's better to keep them together, just like in the current PR. But I do worry that `submittable` isn't very clear for the "before release" case.

Reply:
I really like the name "active". Thanks for the suggestion.